### PR TITLE
[backpack-love] Add changelog link to Readme

### PR DESCRIPTION
**Pull request by matteo-hertel**
The main readme is the place where to look for things, using backpack from the other side I notice how annoying is not  to have that link in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@
 
 https://backpack.github.io/
 
+## Changelog
+
+The up-to-date changelog can be found [here](./CHANGELOG.md)
```
Comment (georgegillams) [inline README.md]: How  about
[View our up-to date changelog](...)

Also full-stops please, darling

Comment (shaundon) [reply README.md]: Sounds good, but make sure to preserve `up-to-date`, as @matteo-hertel had already spelled it correctly 🤓

Comment (georgegillams) [reply README.md]: @matteo-hertel the gif didn't work 😢

**Comment by matteo-hertel** [comment]
@georgegillams @shaundon I've update the copy based on the feedback, if you're merging use squash and merge 🙏

**Pull request merged**